rnd/autogpt_server/autogpt_server/util/service.py

```diff
@@ -16,6 +17,8 @@
 conn_retry = retry(stop=stop_after_delay(5), wait=wait_exponential(multiplier=0.1))
 T = TypeVar("T")
 
+host = os.environ.get("PYRO_HOST", "localhost")
```

Comment:
Read from config `autogpt_server/util/settings.py` so we have a consistent interface for configurables

Reply:
@ntindle I tried to get it to read from Config, but it doesn't read the .env it seems. Only the config.json. So I added it to Secrets which does read from .env. Is there a way to make Config work to read from .env?

Reply:
updated to use Config, since bug fix came in!